paddle/fluid/operators/math/blas.h

```diff
@@ -90,6 +101,28 @@ class Blas {
                    int K, T alpha, const T* A, const T* B, T beta, T* C,
                    int batchCount, int64_t strideA, int64_t strideB) const;
 
+  template <typename T>
+  void MatMul(const framework::Tensor& mat_a, const MatDim& dim_a,
+              const framework::Tensor& mat_b, const MatDim& dim_b, T alpha,
```

Comment:
How are users supposed to retrieve dim_a and dim_b?

If it is something like `dim_a = GetMatDim(mat_a, ...)`, could `MatMul` call this `GetMatDim`; instead of letting users to make the call?

Reply:
The `GetMatDescriptor` can be used in shape inference, data validation, and determine whether to broadcast `batch_size` or not. It is not only be used by MatMul directly.